test/rules/jsx-wrap-multilines/test.tsx.lint

```diff
@@ -0,0 +1,42 @@
+test/rules/jsx-wrap-multilines/test.tsx.lint: Failed!
+Expected (from .lint file)
+Actual (from TSLint)
+const goodSingleLine = <div className="my-class" tabIndex={-1}>{children}</div>;
+
+const goodMultiline = (
+    <div
+        className="my-class"
+        tabIndex={-1}
+    >
+        {children}
+    </div>
+);
+
+const badMultiline = <div
+                     ~~~~~
+    className="my-class"
+~~~~~~~~~~~~~~~~~~~~~~~~~
+    tabIndex={-1}
+~~~~~~~~~~~~~~~~~~
+>
+~~
+    {children}
+~~~~~~~~~~~~~~~
+</div>;
+~~~~~~   [Multiline JSX elements must be wrapped in parenthesis]
+
+const goodSelfClosingMultiline = (
+    <div
+        className="my-class"
+        tabIndex={-1}
+    />
+);
+
+const badSelfClosingMultiline = <div
+                                ~~~~~
+    className="my-class"
+~~~~~~~~~~~~~~~~~~~~~~~~~
+    tabIndex={-1}
+~~~~~~~~~~~~~~~~~~
+/>;
+~~   [Multiline JSX elements must be wrapped in parenthesis]
```

Comment:
how about a case like the following?

```ts
const foo = (<div
    className="bar"
/>);
```

my opinionated stance on this is that a new line should be required after the opening parenthesis and before the closing one, so the above example would be invalid. you'd need to make the error message very clear though